src/app/util/util.js

```diff
@@ -42,3 +42,28 @@ NavPill.propTypes = {
   to: PropTypes.string,
   children: PropTypes.node
 };
+
+/**
+ * Dates
+ */
+import {format} from 'date-fns';
+import * as locales from 'date-fns/locale'; // we are ok with importing all languages
+/**
+ * Call's `date-fns/format` with a locale from `navigator.language`
+ *
+ * @param {Date | number | string} date - a date to format
+ * @param {string?} formatStyle - a format string (eg 'DD.MM.YYY'), default is 'P'
+ * @param {import('date-fns').Locale} locale - date-fns locale object
```

Comment:
Weird way to describe the type of locale

Reply:
If use `Locale` only, an IDE cannot infer the type, which make sense, because where a `Locale` shall come from? Otherwise I need to import type. Shall I do that? Or shall I just use `Locale`?

Reply:
Added `/** @typedef {import('date-fns')} Locale */` let me know if it is ok for you